src/resolvers/index.ts

```diff
@@ -225,6 +255,33 @@ const resolvers = {
 
   // Root resolvers
   Query: {
+    searchGuardianDirectory: async (
+      _: undefined,
+      { search }: { search: string },
+      { dataSources }: PortalUserContext
+    ) => {
+      const { data: searchGuardianDirectory } =
+        await dataSources.personnelAPI.searchGuardianDirectory(search)
+
+      const guardianDirectory: GuardianDirectory[] = []
+      // Pull out and format relevant data to return for directory
```

Comment:
This reuses the code/formatting from the `guardianDirectory` resolver right below it.